src/components/Nav.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import logo from '../Book_Club_Logo.png';
 import '../styles/Nav.css';
 
-const Nav = () => {
+const Nav = ({ location }) => {
+  let navLink;
+  if (location === 'favorites') {
+    navLink = <Link className="landing-page-link" to="/">Return Home »</Link>
```

Comment:
This is a great idea for condition rendering of favorites or back to home button